src/BinCommand.php

```diff
@@ -132,7 +135,7 @@ private function resetComposers(ComposerApplication $application)
     private function chdir($dir)
     {
         chdir($dir);
-        $this->getIO()->writeError('<info>Changed current directory to ' . $dir . '</info>');
+        $this->getIO()->write('<info>Changed current directory to ' . $dir . '</info>', true, IOInterface::VERBOSE);
```

Comment:
Why not leaving it to stderr?

Reply:
it's not an error is it?

Reply:
stderr is not limited to errors, it should also be used for diagnostic / debug output. I think this is also what composer / symfony recommend (cf. https://github.com/symfony/symfony/issues/18744#issuecomment-252883081 for example).

Reply:
Oh I see, well I learned something today! Thanks for the link I'll update the PR shortly